.kokoro/nightly/java11-integration.cfg

```diff
@@ -0,0 +1,37 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-public-resources/java11014"
+}
+
+env_vars: {
+  key: "JOB_TYPE"
+  value: "integration"
```

Comment:
this will need custom args similar to our other integration tests

Reply:
done